### DISCUSSION TITLE
Be sure your fork is named `docs`

### DISCUSSION THREAD

**Discussion by yajo** [OCA/docs#11] [General]
Early adopters: when trying to contribute with decap-cms you might be affected by https://github.com/decaporg/decap-cms/issues/7344 if you already had a fork before the repo was moved to OCA. 

AFAICS this should affect only @dreispt, @florenciafrigieri2, @SirAionTech and @TumbaoJu.

tl;dr: just [rename your fork](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) to `docs`, or [remove it](https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository) and fork again.

**Comment by TumbaoJu** [comment]
@yajo @sbidoul It seems I cannot rename nor delete my fork of the repository created before it was move to OCA. I think I don't have  the access rights to do it. Can it be because it is a fork from OCA and I'm just a member?

PS I followed the instructions to rename but I don't have access to the name field.

**Reply by yajo** [reply]
Could you post a screenshot please? Make sure you're trying to remove or rename https://github.com/TumbaoJu/oca-docs and not https://github.com/OCA/docs.
